### ISSUE TITLE
getting credential EmailAuthProvider

### ISSUE THREAD

**Issue by uttahb** [EddyVerbruggen/nativescript-plugin-firebase#1121]
Is there a way to get the credentials EmailAuthProvider.credential(email, pass) in the the plugin. I tried with both firebaseWebApi and native api, couldn't find the access to the method.

Thanks.

**Comment by EddyVerbruggen** [comment]
Would you mind adding document references for the native iOS and Android SDKs?

**Comment by uttahb** [comment]
Sure
[android](https://firebase.google.com/docs/reference/android/com/google/firebase/auth/EmailAuthProvider)
&
[ios](https://firebase.google.com/docs/reference/swift/firebaseauth/api/reference/Classes/EmailAuthProvider)

**Comment by EddyVerbruggen** [comment]
Thanks, so when you login, a credential is created but never returned to the client because I never thought it would be useful. Can you enlighten me how it can be used?

**Comment by Arystosedes** [comment]
Any news? As can I see that's recommend to use reauthenticateWithCredential when we need to use operations like updatePassword od updateEmail, but to use reauthenticateWithCredential at first we need to get user credential with getEmailAuthProvider for example. Or maybe there is another solution?

**Comment by cupidchan** [comment]
I agree with @plackowski. This information is very helpful for the app. @EddyVerbruggen, can you please suggest a short term "patch" we can do before you officially release the next version with this feature? Thanks!

**Comment by zeejay09** [comment]
Any update on this? Badly needed this for operations like updatePassword.
